Readability.js

```diff
@@ -197,20 +197,29 @@ Readability.prototype = {
       return pathBase + uri;
     }
 
-    function convertRelativeURIs(tagName, propName) {
-      var elems = articleContent.getElementsByTagName(tagName);
-      this._forEachNode(elems, function(elem) {
-        var relativeURI = elem.getAttribute(propName);
-        if (relativeURI != null)
-          elem.setAttribute(propName, toAbsoluteURI(relativeURI));
-      });
-    }
-
-     // Fix links.
-    convertRelativeURIs.call(this, "a", "href");
+    var links = articleContent.getElementsByTagName("a");
+    this._forEachNode(links, function(link) {
+      var href = link.getAttribute("href");
+      if (href) {
+        // Replace links with javascript: URIs with text content, since
+        // they won't work after scripts have been removed from the page.
+        if (href.indexOf("javascript:") === 0) {
+          var text = new Text();
+          text.textContent = link.textContent;
+          link.parentNode.replaceChild(text, link);
```

Comment:
I have a branch that implements `insertBefore` now, and then I read the error log and realized that this wasn't going to help.

Disabling `MutationEvents` helps but then I just get a new error:

  1) Test pages clean-links jsdom "before all" hook:
     TypeError: Cannot call method 'replace' of undefined
      at escapeString (/Users/gkruitbosch/dev/readability/node_modules/jsdom/node_modules/parse5/lib/serialization/serializer.js:26:10)
      at Serializer._serializeTextNode (/Users/gkruitbosch/dev/readability/node_modules/jsdom/node_modules/parse5/lib/serialization/serializer.js:169:56)
      at Serializer._serializeChildNodes (/Users/gkruitbosch/dev/readability/node_modules/jsdom/node_modules/parse5/lib/serialization/serializer.js:78:22)
      at Serializer._serializeElement (/Users/gkruitbosch/dev/readability/node_modules/jsdom/node_modules/parse5/lib/serialization/serializer.js:118:14)
      at Serializer._serializeChildNodes (/Users/gkruitbosch/dev/readability/node_modules/jsdom/node_modules/parse5/lib/serialization/serializer.js:75:22)
      at Serializer._serializeElement (/Users/gkruitbosch/dev/readability/node_modules/jsdom/node_modules/parse5/lib/serialization/serializer.js:118:14)
      at Serializer._serializeChildNodes (/Users/gkruitbosch/dev/readability/node_modules/jsdom/node_modules/parse5/lib/serialization/serializer.js:75:22)
      at Serializer._serializeElement (/Users/gkruitbosch/dev/readability/node_modules/jsdom/node_modules/parse5/lib/serialization/serializer.js:118:14)
      at Serializer._serializeChildNodes (/Users/gkruitbosch/dev/readability/node_modules/jsdom/node_modules/parse5/lib/serialization/serializer.js:75:22)
      at Serializer.serialize (/Users/gkruitbosch/dev/readability/node_modules/jsdom/node_modules/parse5/lib/serialization/serializer.js:60:10)
      at exports.domToHtml (/Users/gkruitbosch/dev/readability/node_modules/jsdom/lib/jsdom/browser/domtohtml.js:16:20)
      at innerHTML (/Users/gkruitbosch/dev/readability/node_modules/jsdom/lib/jsdom/level1/core.js:1409:12)
      at Object.Readability.parse (/Users/gkruitbosch/dev/readability/Readability.js:1745:37)
      at /Users/gkruitbosch/dev/readability/test/test-readability.js:111:34
      at Context.<anonymous> (/Users/gkruitbosch/dev/readability/test/test-readability.js:17:16)
      at callFn (/Users/gkruitbosch/dev/readability/node_modules/mocha/lib/runnable.js:266:21)
      at Hook.Runnable.run (/Users/gkruitbosch/dev/readability/node_modules/mocha/lib/runnable.js:259:7)
      at next (/Users/gkruitbosch/dev/readability/node_modules/mocha/lib/runner.js:268:10)
      at Object._onImmediate (/Users/gkruitbosch/dev/readability/node_modules/mocha/lib/runner.js:289:5)
      at processImmediate [as _immediateCallback] (timers.js:345:15)